cloudant-client/src/test/java/com/cloudant/tests/CloudantClientTests.java

```diff
@@ -136,7 +136,8 @@ public void testUserAgentHeaderIsAddedToRequest() throws Exception {
             //instantiating the client performs a single post request
             CloudantClient client = CloudantClientHelper.newMockWebServerClientBuilder(server)
                     .build();
-            client.executeRequest(createPost(client.getBaseUri(), null, "application/json"));
+            client.executeRequest(createPost(client.getBaseUri(), null, "application/json"))

```

Comment:
True, but we don't run findbugs on the tests.
I've added assertions etc for the return values in 32d6fec.